cfme/web_ui/menu.py

```diff
@@ -121,8 +142,8 @@ def get_rid_of_the_menu_box():
     ),
     ('infrastructure', 'Infrastructure'): (
```

Comment:
By the way, the menu cannot be verified by tests currently, since we have no OS infra support in our automation, although that is being prepared. However I tested it manually and it seems to work. The Control things are fortunately constant with providers so no complex things there, just decision by date.